cmd/influx/global.go

```diff
@@ -0,0 +1,275 @@
+package main
```

Comment:
This file is the renamed `params.go`. It previously contained `getOrgBucketFlags`. Everything else that's added here was moved from `main.go`